CONFIGDOC.md

```diff
@@ -321,6 +323,118 @@ pipeline {
 }
 ``` 
 
+## Register MATLAB as a Jenkins Tool
+To run MATLAB code and Simulink models as part of your automated pipeline of tasks, Jenkins invokes MATLAB as an external program. When you configure your project, you can explicitly specify the MATLAB version that Jenkins should invoke by providing the path to the desired MATLAB root folder. For example, you can use an `environment` block in your `Jenkinsfile` to specify a MATLAB root folder for your Pipeline project.
+
+Instead of specifying the path to the MATLAB root folder on a per project basis, you can register MATLAB as a Jenkins tool, which can then be accessed by any project you configure in Jenkins. To register your desired MATLAB version as a tool, you are required to specify its name and location on the build agent. Once you have registered MATLAB as a tool, you no longer need to specify its root folder path within a project. Jenkins only needs the tool name to access MATLAB.
+
+To register MATLAB as a Jenkins tool:
+
+1) In your Jenkins interface, select **Manage Jenkins > Global Tool Configuration**. A new page opens where you can register different tools with Jenkins.
+2) In the **MATLAB** section of the **Global Tool Configuration** page, locate and click **Add MATLAB**. The section expands and enables you to specify the name and installation location of MATLAB.
+3) specify the name for your desired MATLAB version in the **MATLAB Name** box, and enter the full path to the MATLAB root folder in the **MATLAB root** box. To register MATLAB as a tool, you are not required to select **Install automatically**. 
+4) To confirm your choices, click **Save** at the bottom of the page.
+
+For example, register MATLAB R2020b as a Jenkins tool on your Windows-based agent.

```

Comment:
Probably we can say ' Windows-based Master node ' to emphasize that initial configuration needs to be on the master node.

Reply:
According to MathWorks documentation guidelines, we try to avoid terms such as master/slave. I can use the word "local". That is, Windows-based local agent.

Reply:
How about local Jenkins host or just Window-based Jenkins host

Reply:
Do you see an issue with "Windows-based agent"? Here, I am trying to be consistent with the rest of the documentation. We use agent elsewhere.

Reply:
I dont have a strong openion on this but i feel here we are telling user about where Jenkins is hosted. and agents in general are used to represent remote hosts.

Reply:
Why is "MATLAB Name" in titlecase and "MATLAB root" in sentencecase?

And it looks like the formating in the screenshot is strange.

Reply:
Yes, the UI needs to be updated. 
@abhargav-mw, could you please address Mark's comment?

Reply:
@nbhoski, I reviewed the page. We use "local agent" and "remote agent" consistently throughout the page. To comply with MathWorks guidelines, we use these terms in place of "agent" and "master" (as in https://wiki.jenkins.io/display/JENKINS/Step+by+step+guide+to+set+up+master+and+agent+machines+on+Windows).

Reply:
In the UI 'Name' is displayed by itself. Should we make it start with a lower case? This is how it looks [like]([url](url
![gbl_tool_name_key](https://user-images.githubusercontent.com/68549146/101125389-21917900-361f-11eb-900b-72937bb90385.PNG)
))

Reply:
Thanks, Anmol. I understand now. I was thinking that "MATLAB Name" is the label of the box. What we have is fine and consistent with the rest of our UI.